paket.dependencies

```diff
@@ -16,7 +15,7 @@ group Build
   storage none
   framework auto-detect
 
-  nuget FSharp.Core ~> 4.6.0
+  nuget FSharp.Core
```

Comment:
There was no reason to pin the version of a development dependency. And it prevented Paket from getting the latest FAKE libraries.